CHANGELOG.md

```diff
@@ -2,11 +2,16 @@
 
 The following is a curated list of changes in the Enact agate module, newest changes on the top.
 
-## Unreleased
+## [unreleased]
+
+### Added
+
+- `agate/TabGroup` prop `onTabClick` to handle `onClick` event on it
```

Comment:
onTabClick is not a prop for TabGroup.
it's prop for TabBase. but TabBase is private